src/components/EditInputField.vue

```diff
@@ -3,8 +3,9 @@
         <label>
             {{ fieldLabel }}
         </label>
-        <input v-if="fieldType!=='textarea'" :type="fieldType" v-model="content" @input="handleInput" />
-        <textarea v-if="fieldType==='textarea'" v-model="content" @input="handleInput" />
+        <markdown-editor class='editor' v-if="fieldType==='markdown'" :type="textarea" v-model="content" @input="handleInput" toolbar='' />
```

Comment:
a) This is not set for the description field in the RecipeEdit component. 

b) Also it would be nice to activate the toolbar with some basic functionality (e.g., `bold italic | numlist bullist quote | preview`) to allow users who are not familiar with the Markdown syntax to easily add some formatting to their description.